packages/eslint-config/eslint.config.mjs

```diff
@@ -0,0 +1,122 @@
+import js from '@eslint/js';
+import json from '@eslint/json';
+import eslintConfigPrettier from 'eslint-config-prettier';
+import perfectionist from 'eslint-plugin-perfectionist';
+import react from 'eslint-plugin-react';
+import globals from 'globals';
+import tseslint from 'typescript-eslint';
+//import nlDesignSystemJS from './rules/js.mjs';
+//import nlDesignSystemReact from './rules/react.mjs';
+
+export default tseslint.config(
+  {
+    name: 'nl-design-system/global-ignores',
+    ignores: ['**/dist/', '**/build/', '**/coverage/'],
+  },
+  {
+    name: 'nl-design-system/plugins-settings',
+    plugins: { perfectionist, react },
+    settings: {
+      react: {
+        version: 'detect',
+      },
+    },
+  },
+  {
+    name: '@eslint/js/recommended',
+    files: ['**/*.js', '**/*.cjs', '**/*.mjs', '**/*.jsx', '**/*.ts', '**/*.tsx'],
+    ...js.configs.recommended,
+  },
+  {
+    name: 'typescript-eslint/configs/strict',
+    extends: [...tseslint.configs.strict],
+    files: ['**/*.ts', '**/*.mts', '**/*.tsx'],
+  },
+  {
+    name: 'eslint-plugin-perfectionist/recommended-natural',
+    ...perfectionist['recommended-natural'],
+  },
+  {
+    name: 'eslint-plugin-react/recommended',
+    files: ['**/*.jsx', '**/*.tsx'],
+    ...react.configs.flat.recommended,
+  },
+  {
+    name: 'eslint-plugin-react/jsx-runtime',
+    files: ['**/*.jsx', '**/*.tsx'],
+    ...react.configs.flat['jsx-runtime'],
+  },
+  {
+    // Source code that ends up in a browser
+    name: 'nl-design-system/browser',
+    files: ['**/src/*.ts', '**/src/*.tsx'],
```

Comment:
Moet hier meer bij of is de aanname dat het allemaal TypeScript gaat zijn de juiste?